### PR TITLE
Tests Added

### LINKED ISSUE #22: Automate Tests
- Automating tests for your codebase makes development more predictable and less risky.
- Add tests for existing code every time you change it.
- A simple test to **ALL** API endpoints should be done.

_Help:_ https://docs.pytest.org/en/latest/

### PR DESCRIPTION
**Pull request by diogofalken**
### **Fixes #22**

- Added more dummy entries to help with tests.
- Fixed typo in one of the update functions.
- Were added tests for all the endpoints.
- For GET and DELETE requests, it's just checking if the status code is correct.
- For PUT and POST it's checking the status code and the returned json.
- With this #22 is finished.

In order to test:

1. `https://github.com/diogofalken/bd2-group2.git`
2. `cd bd2-group2`
3. `pytest`

PS: In order to use pytest you need to download it with python package manager.

### CODE REVIEW

**Comment by Guergeiro** [comment]
Thanks. Really good! Though, one test is failing in my environment, could you check it?

image

File: tests/test_a_consumos.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "total": "$9.00"
+}
+
+
+def test_case1_get():
+    response = requests.get(f"{url}{endpointName}/")
+    assert response.status_code == 200
+
+
+def test_case2_get():
```
Comment (Guergeiro) [inline tests/test_a_consumos.py:23]: Test case failing.

image

**Comment by diogofalken** [comment]
Fixed it with my latest commit.
Try it again please.

File: tests/test_a_consumos.py (hunk 1 above)
Comment (Guergeiro) [reply tests/test_a_consumos.py:23]: This should be working now.

File: database/h_dummy.sql [hunk 2]
```diff
@@ -73,7 +73,7 @@ INSERT INTO funcionarios
 VALUES ('PYYAr9K67g', 'BrenoFuncionario', 'x-ypRN-DpQ', TRUE) ;
 
 INSERT INTO consumos
-VALUES ('tykEWk7L-A', NOW(), '_bFWsv-uFg', 'PYYAr9K67g');
+VALUES ('tykEWk7L-A', "Thu, 19 Dec 2019 15:49:54 GMT", '_bFWsv-uFg', 'PYYAr9K67g');
```
Comment (Guergeiro) [inline database/h_dummy.sql]: I can't test this, since I'm not at computer now. But it will most likely fail since PostgreSQL uses `'` and not `"`.

Comment (Guergeiro) [inline database/h_dummy.sql]: I can't test this since I'm not at home. But it will most likely fail since PostgreSQL uses `'` and not `"`.

**Review by Guergeiro — CHANGES_REQUESTED** [review]
Even if it works, please change it to keep consistency.

File: database/h_dummy.sql (hunk 2 above)
Comment (diogofalken) [reply database/h_dummy.sql]: Done.

**Comment by Guergeiro** [comment]
Seems good to me. Will merge this and, in the close future, add it to our automatic tests of Travis CI.

**Review by Guergeiro — APPROVED** [review]
Thanks for the hard work. Will implement this in Travis CI tests tomorrow.

**Pull request merged**